src/goods.ts

```diff
@@ -82,7 +83,11 @@ export function echo(pieces: TemplateStringsArray, ...args: any[]) {
 }
 
 function stringify(arg: ProcessOutput | any) {
-  return arg instanceof ProcessOutput ? arg.toString().trimEnd() : `${arg}`
+  return arg instanceof ProcessOutput
+    ? arg.toString().trimEnd()
+    : arg === ''
+      ? `${''}`
+      : `${arg}`
```

Comment:
Provide a test for this case.

Reply:
I don't think this is a right place for this fix. It should be done in quote function.

Reply:
Hey @antonmedv . When you say quote function, is it the one on core.ts?